mobile-app/app/tailwind.config.js

```diff
@@ -108,6 +108,10 @@ module.exports = {
           500: '#FF9F0A',
           600: '#FFBC54',
           700: '#BF7708'
+        },
+        darkblue: {
```

Comment:
we have a different shading for the colors. Please refer to this: https://www.figma.com/file/XdKJSI9TGYAf1VXJCQf2cZ/DSGN---Foundations?node-id=3%3A360

We can use `blue` and `darkblue` for the shades.


```
blue: {
  50: '#F7FBFE',
  100: '#EFF6FD',
  200: '#D6EAF9',
  300: '#BDDDF5',
  500: '#5AA9E6',
  600: '#5198CF',
  700: '#447FAD',
  900: '#2C5371'
},


darkblue: {
  50: '#242D33',
  100: '#3E627A',
  200: '#4C7895',
  300: '#5F96BA',
  500: '#7FC8F8',
  600: '#A5D9FA',
  700: '#CCE9FC',
  900: '#F2FAFE'
},
```